src/platform.js

```diff
@@ -73,6 +73,24 @@ class App {
     return path.join(this.originalResourcesDir, 'app.asar')
   }
 
+  get commonHookArgs () {
+    return [
+      this.opts.electronVersion,
+      this.opts.platform,
+      this.opts.arch
+    ]
+  }
+
+  /**
+   * @TODO: rename this into something less verbose or remove it altogether
```

Comment:
Let's clean up this TODO before we merge

Reply:
I think this is fine without renaming since it's not part of the public API surface